cmd/soroban-rpc/internal/methods/get_transactions.go

```diff
@@ -134,92 +116,142 @@ func (h transactionsRPCHandler) getTransactionsByLedgerSequence(ctx context.Cont
 			limit = request.Pagination.Limit
 		}
 	}
+	return start, limit, nil
```

Comment:
Maybe a nitpick but it doesn't seem like there's a reason for it to return a pointer since it's always non-nil